src/Cryptol/Backend.hs

```diff
@@ -295,7 +297,7 @@ class MonadIO (SEval sym) => Backend sym where
   bitAsLit :: sym -> SBit sym -> Maybe Bool
 
   -- | The number of bits in a word value.
-  wordLen :: sym -> SWord sym -> Integer
+  wordLen' :: f sym -> SWord sym -> Integer
```

Comment:
I wonder if we should rename `f` to `proxy` to signal the intent that this argument is never used at runtime.